### DISCUSSION TITLE
Feature request - Allowing Regex Search in goto_bookmarks

### DISCUSSION THREAD

**Discussion by ghost** [ahrm/sioyek#1173] [Ideas]
Hey, thanks for the amazing app. I know there is regex search in available in general. I am wondering whether it is possible to do it for goto_bookmarks. I am using bookmarks extensively, sometimes i summarise the article in bookmarks and add some sort of words as tags. With regex search i'd be able to find bookmarks which contain these tags/keywords. Right now i am using alpha version released 2 weeks ago. @ahrm

**Comment by ahrm** [comment]
I will consider this for the next version, you might want to open an issue instead of a discussion because I might forget about the discussion but the list of opened issues is always available.
